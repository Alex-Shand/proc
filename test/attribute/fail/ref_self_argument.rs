struct Something;

impl Something {
    #[proc::attribute]
    fn test(&self) -> proc::TokenStream {
        todo!()
    }
}

fn main() {}
