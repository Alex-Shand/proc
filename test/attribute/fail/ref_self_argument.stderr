error: The attribute function should be free standing
 --> test/attribute/fail/ref_self_argument.rs:5:5
  |
5 |     fn test(&self) -> proc::TokenStream {
  |     ^^
